open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/utils/ConsentRetrievalUtil.java

```diff
@@ -155,42 +153,43 @@ public static boolean validateExpiryDateTime(String expiryDate) throws ConsentEx
      */
     public static void populateDebtorAccount(JSONObject initiation, JSONArray consentDataJSON) {
 
-        if (initiation.get(ConsentExtensionConstants.DEBTOR_ACC) != null) {
-            JSONObject debtorAccount = (JSONObject) initiation.get(ConsentExtensionConstants.DEBTOR_ACC);
-            JSONArray debtorAccountArray = new JSONArray();
+            if (initiation.get(ConsentExtensionConstants.DEBTOR_ACC) != null) {
```

Comment:
Is this indentation is required here in this method?, check below as well.

Reply:
Fixed in [7207d30]